event-timing/gap-pointerdown-pointerup.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<meta charset=utf-8 />
+<title>Event Timing click.</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/resources/testdriver.js></script>
+<script src=/resources/testdriver-actions.js></script>
+<script src=/resources/testdriver-vendor.js></script>
+<script src=resources/event-timing-test-utils.js></script>
+<div id='target'>Click me</div>
+<script>
+  promise_test(async t => {
+    const pointerDowns = []
+    const pointerUps = []
+    blockNextEventListener(window, 'pointerdown', 30)
+    blockNextEventListener(window, 'pointerup', 30)
+
+    // A keydown is used as an interleaving event, to make sure the first event
+    // timing entry was dispatched - await afterNextPaint() is not enough for
+    // all browsers. See discussion in
+    // https://github.com/web-platform-tests/wpt/pull/55577#discussion_r2466579023
+    blockNextEventListener(window, 'keydown', 30)
+    let resolveInterleavingEventDispatched
+    const interleavingEventDispatched = new Promise( resolve => resolveInterleavingEventDispatched = resolve )
```

Comment:
Nit: rather than calling it "interleaving event dispatched" its more correct to say "interleaving event timing reported".

It's pedantic (sorry) but this test is specifically going out of its way *because* of that subtle distinction.

(Feel free to consider this optional feedback since the context is in the comment above)